dts/msm8916-motorola-harpia.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <0x4E 0x81B0>;
+
+	model = "Motorola Mobility MSM 8916 Harpia";
```

Comment:
Change this to something more meaningful (e.g. `Motorola Moto G4 Play`), and make sure it fits on one line when displayed in the fastboot screen.